web/src/app/util/AppBarSearchContainer.js

```diff
@@ -125,3 +116,11 @@ export default function AppBarSearchContainer(props) {
     </SearchContext.Consumer>
   )
 }
+
+SearchProvider.propTypes = {
+  actions: null,
+}
```

Comment:
I don't think this is needed, as `actions` is used in `state` as opposed to coming from a `prop`

Reply:
Removed actions.